ui/components/component-library/avatar-favicon/avatar-favicon.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { BaseAvatar } from '../base-avatar';
+import Box from '../../ui/box/box';
+
+import {
+  BORDER_COLORS,
+  SIZES,
+  DISPLAY,
+  ALIGN_ITEMS,
+  JUSTIFY_CONTENT,
+} from '../../../helpers/constants/design-system';
+
+export const AvatarFavicon = ({
+  size = SIZES.MD,
+  imageSource,
+  className,
+  borderColor = BORDER_COLORS.TRANSPARENT,
+  ...props
+}) => {
+  return (
+    <BaseAvatar
+      size={size}
+      display={DISPLAY.FLEX}
+      alignItems={ALIGN_ITEMS.CENTER}
+      justifyContent={JUSTIFY_CONTENT.CENTER}
+      className={classnames('avatar-favicon', className)}
+      {...{ borderColor, ...props }}
+    >
+      <img
+        className="avatar-favicon__image"
+        src={imageSource || './images/icons/icon-global-filled.svg'}
+        alt="avatar favicon"
```

Comment:
@georgewrmarshall I have passed the fallback URL as a string here. Should I use `onError` prop?

Reply:
Sweet! Done